src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx

```diff
@@ -168,23 +167,46 @@ const UniqueTokenExpandedStateHeader = ({
   const familyMenuConfig = useMemo(() => {
     return {
       menuItems: [
-        !asset?.isPoap && {
-          ...FamilyActions[FamilyActionsEnum.viewCollection],
-        },
-        (asset.external_link || asset.collection.external_url) && {
-          ...FamilyActions[FamilyActionsEnum.collectionWebsite],
-          discoverabilityTitle: formattedCollectionUrl,
-        },
-        asset.collection.twitter_username && {
-          ...FamilyActions[FamilyActionsEnum.twitter],
-        },
-        asset.collection.discord_url && {
-          ...FamilyActions[FamilyActionsEnum.discord],
-        },
+        ...(!asset?.isPoap
+          ? [
+              {
+                ...FamilyActions[FamilyActionsEnum.viewCollection],
+              },
+            ]
+          : []),
```

Comment:
```suggestion
        ...(!asset?.isPoap
          && [
              {
                ...FamilyActions[FamilyActionsEnum.viewCollection],
              },
            ]),
```
This is not working?

Reply:
Hmm, not sure why I changed this, I'll change it back.

Reply:
ts was screaming here because you can have an undefined value inside the array, i think @jxom changed it because of it since this will always give the correct values